application/src/main/java/org/togetherjava/tjbot/commands/TopHelpers.java

```diff
@@ -0,0 +1,114 @@
+package org.togetherjava.tjbot.commands;
+
+import com.github.freva.asciitable.HorizontalAlign;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionMapping;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.CommandData;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+import org.jetbrains.annotations.NotNull;
+import org.jooq.Record3;
+import org.jooq.Result;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.togetherjava.tjbot.db.Database;
+import org.togetherjava.tjbot.util.PresentationUtils;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.jooq.impl.DSL.*;
+import static org.togetherjava.tjbot.db.generated.tables.MessageMetadata.MESSAGE_METADATA;
+import static org.togetherjava.tjbot.util.MessageTemplate.PLAINTEXT_MESSAGE_TEMPLATE;
+
+public final class TopHelpers implements Command {
```

Comment:
This class should use `AbstractCommand` as base class to get rid of the code duplication with the method implementation pattern (as flagged by SonarCloud).

Reply:
Think I saw that but didnt use it as it was under examples. Let me refractor it in